I-DOCSIS 7.0.0 (#2742)
SkylineCommunications/dataminer-docs: release-notes/EPM/I-DOCSIS/EPM_7.0.0_I-DOCSIS.md

---
uid: EPM_7.0.0_I-DOCSIS
---

# EPM 7.0.0 I-DOCSIS

## New features

## Changes

### Enhancements

#### CISCO CBR-8 CCAP Platform: Toggle button to enable/disable polling of OFDM/OFDMA tables [ID_36635]

In some cases, it can occur that the CCAP is not able to retrieve OIDs when polling the OFDM/OFDMA tables, and DataMiner interprets this as a timeout.

> 

Reviewer: Changed and added a little more information

As such, it is now possible to enable or disable the polling of these tables with a new toggle button, so that you can prevent such timeouts.

> 

Reviewer 2: The toggle button is located in the Configuration page, in the second column under the option to disable/enable Debug page.

#### Only active channels associated with cable modem shown [ID_37376]

When you view all channels associated with a cable modem, now only the active channels will be shown instead of all potentially available channels. This will make it easier to focus on the relevant cable modem KPIs.

#### Visual pages now include parent shapes for enhanced network topology [ID_37592]

Alignment errors on the Visual pages for Port, Distribution, and FAT (Forwarding Access Terminal) pages have been corrected. These Visual pages now include parent shapes, making it possible to identify the hub and the Optical Line Terminal (OLT) associated with each entity.

> 

Reviewer: Changed

> 

Reviewer 3: ```suggestion
> [!NOTE]
> This update is related to the GPON solution and has been included in this package because of shared components between solutions.
```

Should this be included as a note, or is this internal?

> 

Reviewer 4: Hi @EdithLansens, please if we can include it as a note.
I am not sure if we need to rephrase it to make it more clear

> 

Reviewer 3: > [!NOTE]
> This change has been implemented in the EPM GPON Solution because of shared components across solutions.

I rephrased it. What do you think, @miguel-slc?

> 

Reviewer 4: Hi @EdithLansens ,
Although this change is related to the EPM GPON solution, it has been added to the EPM I-DOCSIS solution since both solutions currently share some components (Skyline EPM Platform connector)

> 

Reviewer 3: OK! Changing it to:
> [!NOTE]
> This change is related to the EPM GPON Solution because of shared components across solutions.

> 

Reviewer 4: Thanks. It looks ok!

#### Visual enhancements to EPM entity tables [ID_37776]

The column order for the EPM entity tables in the *Below this view* section has been adjusted to align with the column format of the *Data* section for that same entity.

#### Improved performance when navigating to EPM cards through Topology app [ID_38194]

Performance has improved when you navigate to EPM cards through the Topology app. The Visual pages will now load faster and KPIs will no longer appear one at a time.

#### Generic DOCSIS CM Collector now supports both comma and period as decimal separator [ID_38195]

The Generic DOCSIS CM Collector connector can now parse numeric values regardless of whether a comma or period is used as decimal separator.

> 

Reviewer: Changed

#### Generic DOCSIS CM Collector: Improved calculation method for FEC values [ID_38196]

The calculation method for Forward Error Correction (FEC) values in the Generic CM Collector was enhanced. Previously, FEC values were calculated using averages of channel instances, leading to potential discrepancies, particularly in scenarios where Uncorrectable Packet Ratio (UER) was high while the Corrected Packet Ratio (CER) showed as zero. Now, instead of relying on averages, the system aggregates all counters to accurately determine the weight for the selected QAM channel.

> 

Reviewer: Updated language to remove the word "weight" as there was no need to use it.

> The calculation method for Forward Error Correction (FEC) values in the Generic CM Collector was enhanced. Previously, FEC values were calculated using averages of channel instances, leading to potential discrepancies, particularly in scenarios where Uncorrectable Packet Ratio (UER) was high while the Corrected Packet Ratio (CER) showed as zero. Now, instead of relying on averages, the system aggregates all counters to accurately determine the ratio.

Author: ```suggestion
The calculation method for Forward Error Correction (FEC) values in the Generic CM Collector has been enhanced. Previously, FEC values were calculated using averages of channel instances, leading to potential discrepancies, particularly in scenarios where Uncorrectable Packet Ratio (UER) was high while the Corrected Packet Ratio (CER) showed as zero. Now, instead of relying on averages, the system aggregates all counters to accurately determine the ratio.
```

#### Huawei 5688-5800 CCAP Platform connector now aligns CCAP name with element name [ID_38197]

​The Huawei 5688-5800 CCAP Platform connector previously used a custom system name for its EPM CCAP name, which could cause mapping issues when retrieving element data. Now, the connector uses the element name as the CCAP name, resolving problems in Visio when looking up element-specific data.

> #### Skyline EPM Platform: DOCSIS 3.1 OFDM/OFDMA integration [ID_38198]

Author: Can we move this RN to the "New features" section at the top?

Reviewer 4: Hi @MariekeGO, it is not a new feature but a fix.

#### Skyline EPM Platform: Maps now use relational filtering [ID_38199]

Maps now employ relational filtering to identify all available map entities. This ensures a precise alignment with the data presented on the EPM card, enhancing data reliability.

#### Improved OOS reporting: Define thresholds per channel [ID_38425]

Improvements have been made to *Out of Spec (OOS)* reporting by allowing different thresholds for the Rx Power Status, SNR Status, and Post-FEC Status parameters for the US QAM and DS QAM channel modulations. Previously, one threshold was used globally across all channels. Now, thresholds vary depending on the modulation of the associated cable modem (CM) channel. Parameters that used to have fixed thresholds have been replaced with a new table containing the different modulations along with their corresponding thresholds.

> #### Separate procedures for adding a single CCAP/CM pair and creating elements in bulk [ID_38518]

Author: Can we move this RN to the "New features" section at the top?

Reviewer 4: Hi @MariekeGO, yes, it is a new feature. Makes sense to move it to the 'New Features' section

> In DataMiner Cube, the *epm_i_docsis_addnewccapcmpair* pop-up window now includes the *Create Bulk* and *Create Single* options, to either create a single CCAP/CM pair, or create elements in bulk from a CSV file. Additionally, the *Create Single* procedure now includes a step for entering both the get community string and set community string for the collector.

Reviewer 3: ```suggestion
In DataMiner Cube, the *epm_i_docsis_addnewccapcmpair* interactive Automation script now includes the *Create Bulk* and *Create Single* options, to either create a single CCAP/CM pair, or create elements in bulk from a CSV file. Additionally, the *Create Single* procedure now includes a step for entering both the get community string and set community string for the collector.
```

Is this more accurate, @miguel-slc and @EstebanSkyline?

> In DataMiner Cube, the *epm_i_docsis_addnewccapcmpair* pop-up window now includes the *Create Bulk* and *Create Single* options, to either create a single CCAP/CM pair, or create elements in bulk from a CSV file. Additionally, the *Create Single* procedure now includes a step for entering both the get community string and set community string for the collector.

Reviewer 3: ```suggestion
In DataMiner Cube, the interactive Automation script *epm_i_docsis_addnewccapcmpair* now includes the *Create Bulk* and *Create Single* options, to either create a single CCAP/CM pair, or create elements in bulk from a CSV file. Additionally, the *Create Single* procedure now includes a step for entering both the get community string and set community string for the collector.
```

#### Conditional coloring added to dashboard tables [ID_38565]

Conditional coloring has now been implemented across all *CM Overview* and *CM Offline Overview* dashboards. A green color indicates an *OK* parameter status, while a red color indicates an *Out of Spec (OOS)* parameter status.

> #### Skyline EPM Platform: Creating multiple threshold tables for CCAP and CM collectors streamlined [ID_38741]

Author: Can we move this RN to the "New features" section at the top?

To streamline the process of creating multiple threshold tables for CCAP and CM collectors, the possibility to create multiple sets directly from the Upstream and Downstream tables within the platform has been added.

> For this purpose, a new script has also been added, which copies the table from the front end and applies it across active CCAP and CM collectors: *EPM_I_DOCSIS_SetThresholdsTableToCollectors*.

Reviewer 2: For this purpose, the tables were added into a new page called Thresholds Settings. Pages can also be found on the visual Configuration/docsis thresholds settings page.

> For this purpose, a new script has also been added, which copies the table from the front end and applies it across active CCAP and CM collectors: *EPM_I_DOCSIS_SetThresholdsTableToCollectors*.

Reviewer 3: ```suggestion
For this purpose, a new script has also been added, which copies the table from the front-end *Thresholds Settings* page and applies it across active CCAP and CM collectors: *EPM_I_DOCSIS_SetThresholdsTableToCollectors*. These pages can be accessed via the *Configuration* page or *Threshold Settings* page.
```

Is this correct, @sergiosuarezsky?

> For this purpose, a new script has also been added, which copies the table from the front end and applies it across active CCAP and CM collectors: *EPM_I_DOCSIS_SetThresholdsTableToCollectors*.

Reviewer 2: Yesn it is @EdithLansens Thank you

#### Customer configurations showed duplicate entries [ID_38558]

With only Fiber Node channel instances available, duplicate entries could occur in specific customer configurations. To prevent this, new sets of channels have been introduced, linked to the physical representation of interfaces within the CMTSs. With this enhancement, when viewing the channels associated with a node segment, only the physical channels will be displayed, accompanied by their respective KPIs calculated at that level.

#### Firefox browser removed quotation marks from links generated by Microsoft Visio [ID_38651]

The Firefox browser removed quotation marks from links generated by Microsoft Visio, causing display issues within the Citrix environment. This affected the viewing of dashboards when clicking a dashboard link in Cube. To resolve this issue, enhancements have been made to ensure proper dashboard viewing when loading Cube within the Citrix application.

> 

Reviewer: Changed
